packages/storage-plus/src/indexes.rs

```diff
@@ -19,8 +20,8 @@ pub fn index_string(data: &str) -> Vec<u8> {
 
 // Look at https://docs.rs/endiannezz/0.4.1/endiannezz/trait.Primitive.html
 // if you want to make this generic over all ints
-pub fn index_int<T: Endian>(data: T) -> Vec<u8> {
-    data.to_be_bytes().into()
+pub fn index_int<T: Endian>(data: T) -> IntKey<T> {
+    IntKey::<T>::new(data)
```

Comment:
This is fine, but why even use this now that we have K
`U32Key::new(data.age)` should be clear enough (no one needs to think about endian-ness or whatever